controllers/openstackserver_controller_test.go

```diff
@@ -477,8 +463,8 @@ func Test_OpenStackServerReconcileCreate(t *testing.T) {
 			},
 			expect: func(r *recorders) {
 				listDefaultPortsNotFound(r)
-				createDefaultPort(r)
 				listDefaultServerNotFound(r)
+				listDefaultPorts(r)
```

Comment:
Please change this so that it finds no ports. That way we keep testing the same case as before, i.e. port creation.
```suggestion
				listDefaultPortsNotFound(r)
```